lambdas/service/app.py

```diff
@@ -1014,25 +1014,38 @@ def repository_search(entity_type: str,
     termFacets=generic_object_spec
 )
 
+operator_spec = {
+    'is': schema.object(is_=schema.array({})),
+    **{
+        op: schema.object_type(
+            {
+                op: schema.array({},
+                                 items=schema.array({}, minItems=2, maxItems=2),
+                                 minItems=1,
+                                 maxItems=2)
```

Comment:
… and the outer array represent a list of those ranges, why would we only allow one or two of them?

Reply:
@hannes-ucsc  Perhaps setting the cap at 2 outer ranges is too low, do you think there is a good number to set as the upper limit of ranges allowed in a request?

Reply:
lets go with 16 for the maxItems, research if ES has a lower limit.